### DISCUSSION TITLE
Reconocimiento usuarios antiguos

### DISCUSSION THREAD

**Discussion by zebnat** [zebnat/morpho-dimension#16] [Ideas]
# Reconocimiento usuarios antiguos

El usuario de macrojuegos `@luis010213` ha aportado la idea:

![brave_VPlvtCfz8Q](https://user-images.githubusercontent.com/811822/135515623-d93ff8ef-1f31-41e5-b697-17eaabcd27d9.png)

## ¿Cómo sería?

En alguna sección de la aplicación podríamos mostrar una lista de los usuarios más antiguos que más o menos han seguido siendo activos en la comunidad durante todo ese tiempo.

Si alguien quiere aportar más ideas respecto a esta característica, comentad aquí.

**Comment by Darkusito** [comment]
Sisisis, hacer eso con un pequeño apartado donde enseñe alguno de los logros que estos usuarios consiguieron hacer en la página.

(Por ejemplo, recuerdo que luis010213 fué ranking #1 por un largo tiempo)

**Comment by TheMiler** [comment]
Me encanta la idea. Podría ser un lugar donde aparezcan 3 usuarios que estarían en el top y luego abajo los demás usuarios. Con un limite de los mismos que creas conveniente. 
 En la misma lista se mostraría la cantidad de años y meses de registro de cada persona o del top 1-2-3. 
 Esta es una sugerencia de como podría llegar a ser la lista. 
 Un saludo! TheMiler

**Comment by ShiroToroshi** [comment]
Me gusta mucho la idea, como una de la usuaria más antiguas que aún sigue activa trato de ayudar a todo el mundo y los recién llegados.

Att: rosita

**Comment by ghost** [comment]
Me gustaría que a los usuarios antiguos al menos se les diera un tipo de medalla o algo que marque fuera del perfil que son usuarios antigüos
